draft-pauly-oblivious-doh.md

```diff
@@ -132,46 +135,53 @@ via the Oblivious Proxy.
 ~~~
 {: #fig-doh-exchange title="Obvlivious DoH Exchange"}
 
-## HTTP Request {#oblivious-request}
+# RELAY Method
+
+The RELAY method requests that the recipient forward the contents of the
+HTTP request to the destination origin server identified by the request-target
+and forward any response as a result. RELAY is similar to the CONNECT method
+in that the intermediary proxy forwards information between client and target.
+However, RELAY does not require the client to establish an end-to-end connection
+to the target through the proxy, which allows the proxy to re-use an existing
+connection between itself and the target server.
+
+A client sending a RELAY request MUST send the authority form
+of request-target (Section 5.3 of {{!RFC7230}}); i.e., the request-
+target consists of only the host name and port number of the relay
+destination, separated by a colon. For example,
 
-Oblivious DoH queries are created by the Client, and sent to the Oblivious Proxy.
-Requests to the Oblivious Proxy indicate which DoH server to use as an Oblivious
-Target by specifying two variables: "targethost", which indicates the host name
-of the Oblivious Target server, and "targetpath", which indicates the path on which
-the Oblivious Target's DoH server is running. See {{request-example}} for an
-example request.
+    RELAY target.example.com:443 HTTP/1.1
```

Comment:
Are there any implications on the HTTP version that the proxy uses to the target?

Reply:
Hmm, can you elaborate?

Reply:
We should probably be clear either way, but say something like: "the HTTP version that the client uses to communicate with the relay/proxy has no influence on the HTTP version used by the client to communicate with the target".

Reply:
Ah, I see. Yeah, that's what I was thinking. We can clarify.